core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -163,6 +163,7 @@ Hudson.AdministerPermission.Description=\
   This permission grants the ability to make system-wide configuration changes, \
   as well as perform highly sensitive operations that amounts to full local system access \
   (within the scope granted by the underlying OS.)
+Hudson.ConfigureJenkinsPermission.Description=Configure bits of jenkins
```

Comment:
Can we use Jenkins instead of Hudson ?

Reply:
"Configure bits of jenkins" => "Configure Jenkins except for secure risked configuration" ?

Reply:
Good question, i don't rightly know.